src/pydvl/value/shapley/gt.py

```diff
@@ -277,56 +281,66 @@ def reducer(
 
     ###########################################################################
     # Solution of the constraint problem with scipy
+    # DISABLED: highs fails to find solutions that SCS finds
+
+    # A_ub, b_ub = _build_gt_constraints(n, bound=epsilon / (2 * np.sqrt(n)), C=C)
+    # c = np.zeros_like(u.data.indices)
+    # # A trivial bound for the values from the definition is max_utility * (n-1)
+    # bounds = tuple(u.score_range * n)  # u.score_range defaults to (-inf, inf)
+    # result: sp.optimize.OptimizeResult = sp.optimize.linprog(
+    #     c,
+    #     A_ub=A_ub,
+    #     b_ub=b_ub,
+    #     A_eq=np.ones((1, n)),
+    #     b_eq=total_utility,
+    #     bounds=bounds,
+    #     method="highs",
+    #     options={},
+    # )
 
-    A_ub, b_ub = _build_gt_constraints(n, bound=eps / (2 * np.sqrt(n)), C=C)
-    c = np.zeros_like(u.data.indices)
-    total_utility = u(u.data.indices)
-    # A trivial bound for the values from the definition is max_utility * (n-1)
-    bounds = tuple(u.score_range * n)  # u.score_range defaults to (-inf, inf)
-    result: sp.optimize.OptimizeResult = sp.optimize.linprog(
-        c,
-        A_ub=A_ub,
-        b_ub=b_ub,
-        A_eq=np.ones((1, n)),
-        b_eq=total_utility,
-        bounds=bounds,
-        method="highs",
-        options={},
+    ###########################################################################
+    # Solution of the constraint problem with cvxpy
+
+    import cvxpy as cp
```

Comment:
Is there a reason for this import being here?

Reply:
yeah, this was not a dependency before and I was double checking that I was creating the matrices for the linear problem correctly by solving with a different method. I will move it up.